sanic/app.py

```diff
@@ -111,7 +112,7 @@ def decorator(listener):
 
     # Decorator
     def route(self, uri, methods=frozenset({'GET'}), host=None,
-              strict_slashes=False, stream=False, version=None):
+              strict_slashes=None, stream=False, version=None):
```

Comment:
I understand your doubt but it seemed to work here: https://github.com/channelcat/sanic/pull/896

I did the same type of thing a couple days ago when I made that, so should work :)